test/functional/clipboard/clipboard_provider_spec.lua

```diff
@@ -121,7 +121,7 @@ describe('clipboard', function()
   it('invalid g:clipboard shows hint if :redir is not active', function()
     command("let g:clipboard = 'bogus'")
     eq('', eval('provider#clipboard#Executable()'))
-    eq('clipboard: invalid g:clipboard', eval('provider#clipboard#Error()'))
+    eq('g:clipboard is no Dictionary', eval('provider#clipboard#Error()'))
```

Comment:
"invalid" was better. "Is no dictionary" sounds weird.

Reply:
Well, my intention was to give the user a hint on what's wrong. Just "invalid" doesn't seem too helpful.

Reply:
"is no Dictionary" is awkward English and sounds like a mistake. "is not a Dictionary" would be the normal way to phrase it, though I would suggest "g:clipboard must be a Dictionary".